### PR TITLE
chore: remove tracking and export config variable to env file

### PR DESCRIPTION
**Pull request by Blightwidow**
- Remove monitoring and tracking as now open source
- Move censored config to env with example file. This ensure people know that variable to set.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MAPBOX_ACCESS_TOKEN="xyz"
+MAP_SERVICES_API_URL="https://api.mapbox.com"
```
Comment (GitSquared) [inline .env.example]: Map services should point to a [Valhalla](https://github.com/valhalla/valhalla) instance. We could have a public instance as a default to highlight this:

```suggestion
MAP_SERVICES_API_URL="https://valhalla1.openstreetmap.de"
```

File: README.md [hunk 2]
```diff
@@ -25,12 +25,7 @@ If you're curious, feel free to dig around!
 
 This is a basic managed expo app, and there are a few scripts in `package.json` that should help you find your way.
 
-Secrets and API keys have been cut out of this release, and you might need to get new ones and put them in `app.config.ts` to succesfully build the app.
-All removed secrets have been replaced with the `[CENSORED]` tag to easily identify where they're missing.
-
-## Monitoring
-
-The implementation contains leftovers of our monitoring and tracking solutions (UXCam, Sentry). The API keys related to those have been deleted from the source code, and the related accounts on the monitoring platforms have been closed.
+You need to add some secrets by copying the `.env.example` to `.env` and adding you [mapbox access token](https://docs.mapbox.com/help/getting-started/access-tokens/).
```
Comment (GitSquared) [inline README.md]: wording nitpick

```suggestion
You need to provide your own secrets to access the map through [Mapbox](https://docs.mapbox.com/help/getting-started/access-tokens/) and provide support for other map services like snap-to-road path tracing. Copy the `.env.example` file to `.env` and set them up there.
```

File: lib/components/DebugInfo.tsx [hunk 3]
```diff
@@ -39,10 +38,6 @@ export default function DebugInfo() {
 					'unknown'
 				}`}
 				{'\n'}
-				{`build ${Constants.expoConfig?.version ?? 'unknown'}-${(
-					Updates.createdAt ?? new Date()
-				).toISOString()}`}
```
Comment (GitSquared) [inline lib/components/DebugInfo.tsx:44]: this is very useful to have when trying to figure out which version an issue is happening on, I would keep this. It works even without an EAS Updates setup, and in development mode.

**Review by GitSquared — CHANGES_REQUESTED** [review]
thanks for taking a stab!

File: lib/components/DebugInfo.tsx (hunk 3 above)
Comment (Blightwidow) [reply lib/components/DebugInfo.tsx:44]: Added back the version, but without the created at to avoid having `expo-updates` just for this

Comment (GitSquared) [reply lib/components/DebugInfo.tsx:44]: yeah good trade-off, thanks!

**Review by GitSquared — APPROVED** [review]
awesome, tysm!

**Pull request merged**